release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -44,6 +44,19 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-7-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-7-aws-c2s-region"]
+==== Installing a cluster into an AWS C2S region
+
+You can now install a cluster on Amazon Web Services (AWS) into a Commercial Cloud Services (C2S) region. Because C2S regions do not have {op-system} AMIs published by Red Hat, you must upload a custom AMI that belongs to that region. You are also required to include the CA certificates for C2S in the `additionalTrustBundle` field of the `install-config.yaml` file during cluster installation. The installation program does not support destroying a cluster deployed to a C2S region; you must manually remove the resources of the cluster.
+
+//Link for more info when available.
+
+[id="ocp-4-7-gcp-disk-encryption"]
+==== Installing cluster on GCP with disk encryption using a personal encryption key
+
+You can now install a cluster on Google Cloud Platform (GCP) and use a personal encryption key to encrypt both virtual machines and persistent volumes. This is done by setting the `platform.gcp.osDisk.encryptionKey` parameter in the `install-config.yaml` file.
```

Comment:
The field is a bit off. It should be the `controlPlane.platform.gcp.osDisk.encryptionKey`, `compute[0].platform.gcp.osDisk.encryptionKey`, or `gcp.defaultMachinePlatform.osDisk.encryptionKey` field. Also, use the term "field" instead of "parameter".